### DISCUSSION TITLE
Release Date next Minor (5.4)

### DISCUSSION THREAD

**Discussion by chrwell** [nhibernate/nhibernate-core#2779] [General]
Hi there

We are looking forward to the next minor release 5.4 (https://github.com/nhibernate/nhibernate-core/milestone/36), since it'll include a crucial fix for queries we run (namely for this issue: https://github.com/nhibernate/nhibernate-core/pull/2361).

Do you have any idea, when you'll release minor version 5.4?

Thanks for any feedback

**Comment by bahusoid** [comment]
Just use 5.4 development builds https://www.myget.org/feed/nhibernate/package/nuget/NHibernate
First 5.4 release won't be more stable anyway.

**Comment by fredericDelaporte** [comment]
We do not plan release dates. We just do them when there is a consensus it is time to do it.
I am in favor of releasing a 5.4 after a few more additional PR, but we have not discussed this yet.
